core/common/src/thrift/file_system_master.thrift

```diff
@@ -169,7 +174,7 @@ service FileSystemMasterClientService extends common.AlluxioService {
    */
   // TODO(jiri): Get rid of this.
   i64 loadMetadata( /** the path of the under file system */ 1: string ufsPath,
-      /** whether to load meta data recursively */ 2: bool recursive)
+      /** whether to load meta data recursively */ 2: LoadMetadataTOptions options)
```

Comment:
this change breaks backwards incompatible ... the correct way to do this is to add the options as the third argument, and deprecate the second argument

the client should set both the second option and the third option so that it continues to work with servers version 1.0